polkadot/runtime/westend/src/xcm_config.rs

```diff
@@ -142,14 +145,15 @@ impl Contains<Location> for OnlyParachains {
 	}
 }
 
-pub struct CollectivesOrFellows;
-impl Contains<Location> for CollectivesOrFellows {
+pub struct SystemOrFellows<ParaId>(PhantomData<ParaId>);
+impl<ParaId: IsSystem + From<u32>> Contains<Location> for SystemOrFellows<ParaId> {
```

Comment:
Why is this generic over `ParaId`?

Reply:
if we do it this way: https://github.com/paritytech/polkadot-sdk/pull/3768/files#r1533610770
we dont need any generics here